packages/statie/src/Renderable/RenderableFilesProcessor.php

```diff
@@ -65,14 +65,16 @@ public function processFileInfos(array $fileInfos): array
      * @param AbstractGeneratorFile[] $objects
      * @return AbstractGeneratorFile[]
      */
-    public function processGeneratorElementObjects(array $objects, GeneratorElement $generatorElement): array
+    public function processGeneratorElementObjectMetadata(array $objects, GeneratorElement $generatorElement): array
     {
         if (count($objects) === 0) {
             return [];
         }
 
         foreach ($this->getFileDecorators() as $fileDecorator) {
-            $objects = $fileDecorator->decorateFilesWithGeneratorElement($objects, $generatorElement);
+            if ($fileDecorator->getPriority() >= 1000) {
```

Comment:
Why 1000? It feels kinda random number

Reply:
This must be resolve by some other way, e.g. interface that needs to be implemented. Otherwise this breaks when priority of decorators changes